docs/libpod.conf.5.md

```diff
@@ -16,10 +16,10 @@ libpod to manage containers.
   Default OCI runtime to use if nothing is specified in **runtimes**
 
 **runtimes**
-  For each OCI runtime, specify a list of paths to look for.  The first one found is used.
+  For each OCI runtime, specify a list of paths to look for.  The first one found is used. The $PATH environment variable will be searched if no valid executable was found.
```

Comment:
Searched for what?  Are you hard coding it to search for runc?

Reply:
I rephrased it a bit to make it more clear, hope this is fine now :)